client/src/components/scanHostList/scanHostList.js

```diff
@@ -102,15 +80,14 @@ class ScanHostList extends React.Component {
 
     getInspectionScanResults(scanId, queryObject)
       .then(results => {
-        const morePages = !usePaging && _.get(results.value, 'data.next') !== null;
+        const morePages = fetchAll && _.get(results.value, 'data.next') !== null;
 
         this.setState({
           inspectionScanResultsPending: morePages,
           scanResults: this.addResults(results),
-          prevResults: morePages || this.state.connectionScanResultsPending ? this.state.prevResults : [],
-          disableNext: !usePaging || _.get(results.value, 'data.next') === null,
-          disablePrevious: !usePaging || _.get(results.value, 'data.previous') === null
+          prevResults: morePages || this.state.connectionScanResultsPending ? this.state.prevResults : []
         });
+        this.loading = this.state.connectionScanResultsPending;
```

Comment:
Note: we probably need to start looking at being consistent, for tracking, and continue declaring class level properties in the constructor. they tend to get lost a little